src/utils/metakg/path_finder.py

```diff
@@ -1,9 +1,11 @@
-import networkx as nx
 import logging
+
+import networkx as nx
+
 from controller.metakg import MetaKG
 from model import ConsolidatedMetaKGDoc
 
-logger=logging.basicConfig(level=logging.INFO, filename="missing_bte.log")
+logging.basicConfig(level="INFO")
```

Comment:
I think we should remove this line here. Typically we don't want to set it to a fixed value in our code.